frontend/src/Login/authReducer.js

```diff
@@ -0,0 +1,14 @@
+import { LOGIN } from './actionCreator';
+
+const defaultState = {
```

Comment:
please call it initialState

```suggestion
const initialState = {
```